test/unit/Libify.js

```diff
@@ -0,0 +1,308 @@
+import React from 'react';
+import {xdr, Transaction, Operation} from 'stellar-sdk';
+import Libify from '../../src/utilities/Libify';
+
+describe('Libify-built', () => {
+  describe('simple transaction', () => {
+    let simpleTransaction = Libify.buildTransaction({
+      sourceAccount: 'GCNZ6DLRLBDUUTXMGKHMCIG44I6VSZ4U6ACY6CSICK5DFVRATYJTAEVT',
+      sequence: '321',
+      fee: '42',
+      memoType: '',
+      memoContent: ''
+    }, [
+      {
+        id: 0,
+        attributes: {
+          destination: 'GDCIK56SS6B3WEIY2GVPNI5GA7SLXVDUMLAPJBCVD3MYVUJQX2OLIFBX',
+          startingBalance: '30'
+        },
+        name: 'createAccount'
+      }
+    ]);
+    let decodedTx = new Transaction(simpleTransaction.xdr);
+
+    it('contains specified source account', () => {
+      expect(decodedTx.source).to.equal('GCNZ6DLRLBDUUTXMGKHMCIG44I6VSZ4U6ACY6CSICK5DFVRATYJTAEVT');
+    });
+
+    it('contains specified sequence number', () => {
+      expect(decodedTx.sequence).to.equal('321');
+    });
+
+    it('contains specified fee', () => {
+      expect(decodedTx.fee).to.equal(42);
+    });
+
+    it('contains no memo', () => {
+      expect(decodedTx.memo._switch.name).to.equal('memoNone');
+    });
+
+    it('contains one operation', () => {
+      expect(decodedTx.operations.length).to.equal(1);
+    });
+  })
+
+  describe('large transaction', () => {
```

Comment:
This can be better as a call to `context`, not `describe`.  While the two functions are aliases, they have an important distinction for communicating intent to other coders:  `describe` is used to specify the thing that is being tested (in this case `Libify.buildTransaction`) , context is used to specify the context in which you test is operating (i.e. the input or environment).